presto-docs/src/main/sphinx/release/release-0.203.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.203
+=============
+
+General Changes
+---------------
+
+* Fix spurious duplicate key errors from :func:`map`.
+* Fix planning failure when correlated subqueries containing a ``LIMIT``
+  clause are used within ``EXISTS`` (:issue:`10696`).
+* Fix out of memory error caused by missing pushback checks in data exchanges.
+* Fix execution failure for queries with cross joins when using bucketed execution.
```

Comment:
maybe "Fix execution failure for queries containing a cross join when using bucketed execution."?